src/colors.js

```diff
@@ -0,0 +1,114 @@
+import React from 'react'
+import { Box } from 'theme-ui'
+
+const InlineColor = ({ sx, color, children, ...props }) => {
+  return (
+    <Box
+      as='span'
+      sx={{ display: 'inline-block', color: color, ...sx }}
+      {...props}
+    >
+      {children}
+    </Box>
+  )
+}
+
+const Colors = {}
+
+Colors.Primary = ({ sx, children, ...props }) => {
+  return (
+    <InlineColor color='primary' {...props}>
+      {children}
+    </InlineColor>
+  )
+}
```

Comment:
Ooh `sx` is currently not getting passed to `InlineColor`. Could even just do
```suggestion
Colors.Primary = (props) => {
  return (
    <InlineColor color='primary' {...props} />
  )
}
```

Reply:
done!